src/App.js

```diff
@@ -1,16 +1,44 @@
-import "./App.css";
+import React, { createContext, useReducer } from 'react'
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import Home from "./pages";
 import ContactMePage from "./pages/contactme";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { IntlProvider } from 'react-intl'
+import { messages } from './translate'
+import "./App.css";
+
+export const LangContext = createContext();
```

Comment:
you can take this context, reducer etc to some other file and import it where neededs